### DISCUSSION TITLE
Maui Version

### DISCUSSION THREAD

**Discussion by developer9969** [smstuebe/xamarin-fingerprint#218] [Q&A] [answered]
Hi 
Is there any plan soon to port this wonderful plugin to MAUI?
Any preview we could use ?
thanks

**Comment by smstuebe** [comment, marked as answer]
I'm currently working on it. It's already working in the maui demo app and a preview will be released soon™ 
https://github.com/smstuebe/xamarin-fingerprint/tree/maui-support

**Reply by developer9969** [reply]
Excellent !!

**Reply by smstuebe** [reply]
@developer9969 I did a quick release of the yet to be tested version.
Feel free to try it out and give feedback. Especially if it installs correctly in your projects.
https://www.nuget.org/packages/Plugin.Fingerprint/3.0.0-beta.1

**Reply by andreas-nesheim** [reply]
Installs correctly in my .MAUI project. Also works on both Android emulator and iOS simulator.

**Reply by developer9969** [reply]
Same works on both both Android emulator and iOS simulator

**Reply by JaredP94** [reply]
Hi @smstuebe,

Initial testing has worked well for me as well!

I did notice that I am getting warnings about this functionality only being supported on Android SDK 31+. Is this intentional?

P.S appreciate your efforts to support MAUI early :)

![image](https://user-images.githubusercontent.com/13906915/170363920-eed39ff0-5a3c-4fc6-9b79-24f370ea1f34.png)

**Reply by WillCr2** [reply]
Is it possible to fingerprint with blazor in maui?

**Reply by Welchen** [reply]
@smstuebe is this still being worked on?

**Reply by leungkimming** [reply]
Basically it works in Android and IOS but in IOS, the fingerprint dialog Title was ignored and always displayed the Project's Application Title.


    <ApplicationTitle>**MauiSample**</ApplicationTitle>

var dialogConfig = new AuthenticationRequestConfiguration("**My App**", reason) {
   CancelTitle = cancel,
   FallbackTitle = fallback,
   AllowAlternativeAuthentication = swAllowAlternative.IsToggled,
   ConfirmationRequired = swConfirmationRequired.IsToggled
};

dialogConfig.HelpTexts.MovedTooFast = tooFast;

var result = await Plugin.Fingerprint.CrossFingerprint.Current.AuthenticateAsync(dialogConfig, _cancel.Token);

In IOS, "My App" was not displayed but the "MauiSample" was displayed.

**Comment by MichaelShapiro** [comment]
This is an awesome plugin and I currently use 3.0.0 beta1 in maui app (iOS and Android). It installs correctly and works in the project without any issues so far. My question is - @smstuebe, are you still working on it and do you plan to move it out of beta and into production?  Thank you so much for this plugin.

**Comment by GDT48** [comment]
I have successfully managed to implement the plugin in .net7 Maui but needed to downgrade from 3.0.0 to 2.1.5.
Question I have is this works well on Android, but my project is cross-platform.  When I run the project in Widows I get an error which seems to indicate the Nuget pacage is not compatable.  Is there any way to disable/ignore the plugin when running on the Wndows platform?
